src/style.css

```diff
@@ -1,3 +1,85 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url('Roboto-Medium.ttf');
+}
+
+html {
+  --brand-color: #00acdc;
+
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  text-transform: uppercase;
+}
+
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100dvw;
+  height: 60px;
+  background-color: #fff;
+  padding: 0 50px;
+}
+
+.logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+ul {
+  list-style: none;
+  display: flex;
+  height: 100%;
+}
+
+li {
+  position: relative;
+  height: 100%;
+  margin-right: 20px;
+  display: flex;
+  align-items: center;
+}
+
+li:last-child {
+  margin: 0;
+}
+
+a {
+  display: flex;
+  text-align: center;
+  align-items: center;
+  text-decoration: none;
+  color: black;
+  height: 100%;
+}
+
+a:hover {
+  color: var(--brand-color);
+}
+
+a.is-active::after {
```

Comment:
```suggestion
is-active::after {
```

Reply:
This mistake was removed, using only .is-active class selector for this.